apis/applicationsets/v1alpha1/status.go

```diff
@@ -52,21 +52,24 @@ type ApplicationSetConditionStatus string
 type ApplicationSetConditionType string
 
 type ResourceStatus struct {
-	Group           string        `json:"group,omitempty" protobuf:"bytes,1,opt,name=group"`
-	Version         string        `json:"version,omitempty" protobuf:"bytes,2,opt,name=version"`
-	Kind            string        `json:"kind,omitempty" protobuf:"bytes,3,opt,name=kind"`
-	Namespace       string        `json:"namespace,omitempty" protobuf:"bytes,4,opt,name=namespace"`
-	Name            string        `json:"name,omitempty" protobuf:"bytes,5,opt,name=name"`
-	Status          string        `json:"status,omitempty" protobuf:"bytes,6,opt,name=status"`
-	Health          *HealthStatus `json:"health,omitempty" protobuf:"bytes,7,opt,name=health"`
-	Hook            bool          `json:"hook,omitempty" protobuf:"bytes,8,opt,name=hook"`
-	RequiresPruning bool          `json:"requiresPruning,omitempty" protobuf:"bytes,9,opt,name=requiresPruning"`
-	SyncWave        int64         `json:"syncWave,omitempty" protobuf:"bytes,10,opt,name=syncWave"`
+	Group                        string        `json:"group,omitempty" protobuf:"bytes,1,opt,name=group"`
+	Version                      string        `json:"version,omitempty" protobuf:"bytes,2,opt,name=version"`
+	Kind                         string        `json:"kind,omitempty" protobuf:"bytes,3,opt,name=kind"`
+	Namespace                    string        `json:"namespace,omitempty" protobuf:"bytes,4,opt,name=namespace"`
+	Name                         string        `json:"name,omitempty" protobuf:"bytes,5,opt,name=name"`
+	Status                       string        `json:"status,omitempty" protobuf:"bytes,6,opt,name=status"`
+	Health                       *HealthStatus `json:"health,omitempty" protobuf:"bytes,7,opt,name=health"`
+	Hook                         bool          `json:"hook,omitempty" protobuf:"bytes,8,opt,name=hook"`
+	RequiresPruning              bool          `json:"requiresPruning,omitempty" protobuf:"bytes,9,opt,name=requiresPruning"`
+	SyncWave                     int64         `json:"syncWave,omitempty" protobuf:"bytes,10,opt,name=syncWave"`
+	RequiresDeletionConfirmation bool          `json:"requiresDeletionConfirmation,omitempty" protobuf:"bytes,11,opt,name=requiresDeletionConfirmation"`
```

Comment:
This is already part of the spec. Do we need it to be in the status as well?

Reply:
I've reverted all these changes to bring the PR to a minimum of changes. Thanks for the comment!